shared/src/simulation/events/transfer-patients-in-specific-vehicle-request.ts

```diff
@@ -32,11 +35,13 @@ export class TransferPatientsInSpecificVehicleRequestEvent
     constructor(
         patientIds: UUIDSet,
         vehicleId: UUID,
+        transferInitiatingRegionId: UUID | undefined,
```

Comment:
```suggestion
        transferInitiatingRegionId?: UUID,
```

For consistency with the class member.